config/credentials.yml.enc

```diff
@@ -1 +1 @@
-LWyDo1wOxhizYbWCfQM0q8MLzo3qGS0FOMyraLfteOzQdIQgTPTqHfrIOwtDerzBA4TsGMpYtVS0HK8mgqNRW4NFg4l6+EtpgFtZ1+hc5xAdUp3IvFxTxWRfVUf1FdZfqhhR6Jnx/qFNHgwClyLqwsLa6COI2w3stUYZN5VcgdNxBjgz5e2ZINO/iQOqS7Hg0E77Tp6wmZ/TWexbDrNqHcf5DrfCKPFnGPmJ7jlOb2ioBRNHWA655cGX2TsR2qkT5REfFxvHpTqJhEOIykLU0AX15KaSdLvqhnC3ZFaOAZ131Vb3OU/kg0x6GeHJsIuv+NfNxEsRYclj5lTptDkP5YW1BfYJjPbD1tKTyu6J2CKMJymEQrOt59qVttjYtwqnUwmZh9bunge8t9kdwmf1s2AsgNEsB36ATaDj--dB83ekh/0oHgWRl8--Qwvb0VEtn2r9nZnU+Oiu1Q==
\ No newline at end of file
+9bf0N4PssFlbp+5owQg3cep8nogj1mIErPCSTXTyDdpCqUqC7iIBmsjlV6ODUQ06ZqSNHtLnLyHfBeOG6wf9jnDhtKxOlROumt2wH2zTJ6IVlEg5DViv466dtpGnSax1r04pFMOJYRFWDPr4k8jqsjUjQnQYJFFUF5GgPx82axx/JpBX0WdjCxjAHtBEJkL21bEru+F3cr957rpj0e6/e6KE/ccHBFxs706ZO+PX3htc36W4FpAImhR5i515BI5gPz4+tpxpPIntk45vTVS7PhLA5IoYmLlJ0Iqw5S5OiOLxqaNkHFRuX+qGaSsJtlArf+LeontVKeQXAqRDAFCXnoTNcTE8mRg1AZPV7CSV4/1ZP2ba+UlgnU+2zoH5a9QHtu+ablxWvc+0Bi0hhiZf8Th3ExO6jmoUimEk--f+zJHAqs2UqqRqfh--Y8/8gu0hRbfJFueHVS/70w==
```

Comment:
Creds in a repo?

Reply:
yes, that was in the repo alreayd and it is no issue, the secret is not in the repo
for more info check [here](https://blog.saeloun.com/2019/10/10/rails-6-adds-support-for-multi-environment-credentials.html#background)

Reply:
Fair enough. It still seems risky as an accidental exposure of the key in the repo exposes the creds. Generally I'd expect credentials to be handled as configuration which would be specific to a reference implementation and not part of a code base.